.github/workflows/ci.yml

```diff
@@ -193,21 +187,9 @@ jobs:
           projectName: ember-toucan-core
           directory: ./docs-app/dist
           gitHubToken: ${{ secrets.GITHUB_TOKEN }}
-
-  PostPreviewURLascommenttoPR:
-    name: Post Preview URL as comment to PR
-    runs-on: ubuntu-latest
-    needs: PublishDocstoCloudflarePages
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          persist-credentials: false
-      - uses: ./.github/actions/pnpm
-      - uses: ./.github/actions/download-built-package
       - uses: marocchino/sticky-pull-request-comment@v2
         with:
           message: |+
             ## Preview URLs
-            GH Env: ${{ needs.PublishDocstoCloudflarePages.outputs.env }}
-            docs: ${{ needs.PublishDocstoCloudflarePages.outputs.url }}
-#            api docs: ${{ needs.PublishDocstoCloudflarePages.outputs.url }}/api/modules.html
```

Comment:
Went ahead and deleted this comment as we aren't doing these yet (but also no plan to soon™).